package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "imagekit-media-library-widget",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
`package-lock.json` is oudated, please run `npm install` after changing the version